.travis.yml

```diff
@@ -13,11 +13,12 @@ addons:
   homebrew:
     update: true
     packages:
-      - ninja
+      - python@3
 
-cache:
-  directories:
-    - ./packages
+install:
+  - export PATH=$(brew --prefix llvm)/bin:$PATH
 
 script:
-  - ./codebuild/common-macos.sh -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+  - export CODEBUILD_SRC_DIR=$(pwd)
+  - python3 -c "from urllib.request import urlretrieve; urlretrieve('https://raw.githubusercontent.com/awslabs/aws-c-common/master/codebuild/builder.py', 'builder.py')"
```

Comment:
Would it make sense to tie this to the release/submodule of common  we're synced with?

Reply:
that would make sense for aws-crt-python, but it would be inconsistent with how it's working in all the other repos.